.toys/ci.rb

```diff
@@ -309,7 +309,7 @@ def run_toplevel
     puts "toplevel: bundle ...", :bold, :cyan
     result = exec ["bundle", @bundle_task, "--retry=#{bundle_retry}"]
     unless result.success?
-      @errors << [dir, "bundle"]
+      @errors << [@cur_dir, "bundle"]
```

Comment:
Whoops, thanks for finding this. Best to use the literal string `"toplevel"` (as is used a few lines down) instead of `@cur_dir` since we want the error message to note that the toplevel bundle failed, whereas `@cur_dir` is whatever directory we were in when CI was invoked, which may not be the toplevel directory.

Reply:
Thanks! Pushed new changes.